src/Service/DTO/SponsorshipLevelBenefitData.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service\DTO;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+final class SponsorshipLevelBenefitData
+{
+    private $id;
+
+    /**
+     * @Assert\NotNull()
+     */
+    private $sponsorshipLevel;
+
+    /**
+     * @Assert\NotNull()
+     */
+    private $sponsorshipBenefit;
+
+    /**
+     * @Assert\Type(type="string")
+     */
+    private $content;
+
+    public function __construct(?SponsorshipLevelData $sponsorshipLevel = null, ?SponsorshipBenefitData $sponsorshipBenefit = null, ?string $content = null, ?string $id = null)
```

Comment:
Oh! This is a lot of optionals/nullables. What is this actually supposed to do? You're not sure you need dependencies? How come?